woocommerce/woocommerce-gutenberg-products-block
# Restructure Product Collection editor files

## What

Part of changes from https://github.com/woocommerce/woocommerce-blocks/pull/10952:
- move components and files related to Product Collection's Editor experience under the `/edit` directory
- move Inner Blocks structure to `constants` and create additional methods (required in the scope of https://github.com/woocommerce/woocommerce-blocks/pull/10952).

This PR doesn't add any new functionality, it's purely for restructuring purpose.

## Why

Improve navigation and cleanup Product Collection block directory with more changes coming.

## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Go to Editor
2. Add Product Collection block
3. Make sure there are no console errors
4. Smoke test block by changing some of the filters in Editor
5. Save and go to frontend
6. Make sure there's no errors


## Screenshots or screencast



| Before | After |
| ------ | ----- |
|        |       |

## WooCommerce Visibility


Required:


## Checklist

Required:


Conditional:


## Changelog


> Add suggested changelog entry here.
